npm/css-src/sass/_success_summary.scss

```diff
@@ -0,0 +1,105 @@
+.success-summary {
+  border: 5px solid #00703c;
+  color: #0b0c0c;
+  margin-bottom: 30px;
+  padding: 15px; }
+@media print {
+  .success-summary {
+    color: #000; } }
+@media (min-width: 40.0625em) {
+  .success-summary {
+    padding: 20px; } }
+@media (min-width: 40.0625em) {
```

Comment:
Any idea why these rules aren't combined, given that they have the same min-width?

Reply:
This one and the one below is because I copied the css and not the sass. It should be better now.